README.md

```diff
@@ -15,11 +30,51 @@ Current status:
 
 See the [docs](https://github.com/MastodonC/witan.models.demography/blob/master/doc/intro.md) for more info about the methodology.
 
-## Development tools
+
+## Contribute to `witan.models.demography`
+
+### General practices
+
+If you wish to contribute to `witan.models.demography`, please follow read the following guidelines.
+
+#### Github guidelines
+* **Fork this repository** (or clone it if you have writing writes to the reposotory).
+* **Create a new branch**. Let's try and keep the following naming conventions for branches:
+  * `feature/<what-this-branch-solves>`, example: `feature/exception-logging`
```

Comment:
Its a small thing, and feel free to not do it, but could you make this a model type of feature? e.g. 'feature/project-mortality-rates' or something